xbmc/ServiceManager.cpp

```diff
@@ -177,7 +177,10 @@ bool CServiceManager::InitStageThree(const std::shared_ptr<CProfileManager>& pro
     *profileManager));
 
   m_contextMenuManager->Init();
-  m_PVRManager->Init();
+
+  // Init PVR manager after login, not already on login screen
+  if (!profileManager->UsingLoginScreen())
```

Comment:
Does this cause `m_PVRManager->Init();` to never be called if login screen is in use?

Reply:
In this case it will be called from `CProfileManager::FinalizeLoadProfile()`